### PR TITLE
Feature | Update notification mailer

### PR DESCRIPTION
**Pull request by EdgarB**
## Description
Add notification mailer template without assets.

## How has this been tested?

Please mark the tests that you ran to verify your changes. If difficult to test, consider providing instructions so reviewers can test.

- [x] Manual testing
- [ ] System tests
- [ ] Unit tests
- [ ] None

## Checklist

- [ ] CI pipeline is passing
- [ ] My code follows the conventions of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if applicable)
- [ ] I have added seed data to the database (if applicable)

## Release tasks

Add any tasks that need to be done before/after the release of this feature.

## Screenshots/Loom
image

### CODE REVIEW

File: app/views/notifications_mailer/session_notification.html.erb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<div class="container">
+  <% starts_in = minutes_between(DateTime.now, @session.starts_at) %>
+  <% if starts_in > 1  && starts_in < 60 %>
+    <h1 class="title">Your bookmarked session is starting in <%= starts_in %> minutes </h1>
+  <% elsif starts_in <= 1 %>
+    <h1 class="title">Your bookmarked session is starting now</h1>
+  <% else %>
+    <h1 class="title">Your bookmarked session is starting soon </h1>
+  <% end %>
```
Comment (AlanSoto31) [inline app/views/notifications_mailer/session_notification.html.erb:9]: The `title` method in the `SessionReminderNotifier` class might be useful for defining the title. If it doesn't fit the purpose, I suggest extracting this logic into the `SessionReminderNotifier` class to maintain consistency.

**Review by AlanSoto31 — CHANGES_REQUESTED** [review]
The email looks odd

image

**Review by AlanSoto31 — CHANGES_REQUESTED** [review]

**Review by AlanSoto31 — APPROVED** [review]
Great work!

**Review by Sergio-e — CHANGES_REQUESTED** [review]
Don't merge. No changes are needed for the moment, I'll post them soon

**Review by Sergio-e — APPROVED** [review]

**Pull request merged**